app/views/my_facilities/blood_pressure_control.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="row">
+  <div class="col-lg-12 pr-lg-0">
+    <%= render "shared/my_facilities/blood_pressure_control" %>
```

Comment:
Do we need a partial for a single render?

Reply:
Ah I see how the partials are the "graphs", and the view controls how to lay them out on a page. I like it.